server/schedule/opts.go

```diff
@@ -17,6 +17,10 @@ import (
 	"time"
 )
 
+// Simulating is an option to overpass the impact of accelerated time. Should
+// only turned on by the simulator.
+var Simulating bool
```

Comment:
I don't like the way using a global var. Maybe you can add a function like IsSimulating instead.

Reply:
I'm not quite understand your suggestion. Where should the function be defined and where should we set it?

Reply:
Maybe in the cluster, you set it in the simulator.

Reply:
Hi @siddontang , I would like not to do it.
It is doable to add a flag to `server.Config` and then pass the flag all around, but will involve a lot of adjustments to the components that do not need to know whether it is simulating or not.
In fact, only the code in the `schedule` and `schedulers` needs to be concerned with whether it is simulating. And the effect is global, so I think it's fine and simple to use to global flag.

Reply:
I see that you only use this in BasicCluster, is it hard to pass from outside?